app/models/invenio_rdm_record_converter.rb

```diff
@@ -133,42 +135,50 @@ def creators(creators)
       # Galter Health Sciences Library is a User on production DigitalHub, but it does NOT have a formal_name value
       dh_user = User.find_by(formal_name: creator)
 
-      # organization
+      # Organization
       if organization?(creator)
-        name = creator
-        creator_type = "organisational"
-        family_name = ""
-        given_name = ""
-        identifiers = {}
-        affiliations = [] # only used for creator_type "personal"
-      # user
+        {
+          "person_or_org":
+            Hash.new.tap do |hash|
+              hash["name"] =  creator if creator.present? && !creator.include?("not identified")
+              hash["type"] = "organisational"
+            end
+        }.with_indifferent_access
+      # User within DigitalHub
       elsif dh_user.present?
-        name = dh_user.formal_name # will always be this regardless of type
-        creator_type = "personal"
-        dh_user_display_name = dh_user.display_name.split(' ') # split name into components to be reused
-        family_name = dh_user_display_name.pop # remove last value from display name
-        given_name = dh_user_display_name.join(' ') # the remaining strings becomes given name
-        identifiers = dh_user.orcid.present? ? {"orcid": dh_user.orcid.split('/').pop} : {}
-        affiliations = [] # TODO: user.affiliation is nil for all users on production... where do we get this value?
-      # TODO: unknown personal, what do we enter for these fields?
+        dh_user_formal_name = dh_user.formal_name.split(',') # split name into components to be reused
+        family_name = dh_user_formal_name.shift # remove first value from formal name
```

Comment:
Mentioned this in the spec, but formal name is really the proper field to use here instead of display name.

Reply:
Maybe this is in a release note somewhere, but is `name` subfield removed? if so, please ignore my other comments about adding it.

Reply:
Oh, I missed this top comment while going through the review. Mentioned it in other comments, but if the type is personal you do not use the name field.